tests/test_api.py

```diff
@@ -1908,3 +1909,106 @@ def test_do_create_prod_build_branch_required(self, osbs, branch_name):
                                            inner_template=inner_template,
                                            outer_template=outer_template,
                                            customize_conf=customize_conf)
+
+    @pytest.mark.parametrize('lowpriority', ['yes', None])
+    def test_create_build_config_low_priority(self, caplog, lowpriority):
+        # Tests temporarily removing lowpriority nodeselector
+        with NamedTemporaryFile(mode='wt') as fp:
+            fp.write(dedent("""\
+                [general]
+                build_json_dir = inputs
+                """))
```

Comment:
Don't we want a `[default]` section with a `low_priority_node_selector` item configured?